airbyte-webapp/src/views/Connector/ConnectorDocumentationLayout/ResourceNotAvailable/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { faArrowDown } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { FormattedMessage } from "react-intl";
+
+import { Button } from "components/ui/Button";
+import { Text } from "components/ui/Text";
+
+import { SourceDefinitionRead } from "core/request/AirbyteClient";
+import { useDocumentationPanelContext } from "views/Connector/ConnectorDocumentationLayout/DocumentationPanelContext";
+
+import styles from "./ResourceNotAvailable.module.scss";
+import octaviaWorker from "../octavia-worker.png";
+import { useAnalyticsTrackFunctions } from "../useAnalyticsTrackFunctions";
+
+export const ResourceNotAvailable: React.FC<
+  React.PropsWithChildren<{
+    activeTab: "erd" | "schema";
+    isRequested: boolean;
+    setRequested: (val: boolean) => void;
+  }>
```

Comment:
Please make this a `interface ResourceNotAvailableProps`, which we usually always do whenever there's more than one property.